### ISSUE TITLE
Add real world examples

### ISSUE THREAD

**Issue by theKashey** [zerobias/effector#14]
Examples - is the best way to understand the library.
Some examples already could be found in readme or in tests(🤓) but could be better.

**Comment by DQvsRA** [comment]
Yes, since the library claimed to be adopted for Vue I would like to see examples for Vue, especially for complex applications, to be able to compare it with brilliantly designed Vuex and Mobx. Subscribed.

**Comment by maximelebreton** [comment]
Yes, not having a real world example for `Vue` stops immediatly my interest.
I wanted to test it for a current Vue App, but without documentation or example, I need to go deep in the code to understand how this library works, and I have no time for this.
Please provide a working example, even small, and give to your potential users the ability to test and understand it quickly.

Thanks!

**Comment by zerobias** [comment]
@maximelebreton [real application](https://github.com/ilajosmanov/braces-effector) with  vue and effector

**Comment by maximelebreton** [comment]
thanks @zerobias!

**Comment by smenshikov** [comment]
Probably this issue can be fixed because the documentation now is better
